docs/_documentations/mdt-eclipse-installinfo.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title: Installing Codewind for Eclipse
+description: Installing Codewind for Eclipse
+keywords: install, installing, add, adding, tools, Eclipse, installing Codewind for Eclipse
+duration: 1 minute
+permalink: mdt-eclipse-installinfo
+type: document
+order: 
+parent: 
+---
+
+# Installing Codewind for Eclipse
+
+The Codewind installation includes two parts.
+1. The first part installs when you Install Codewind from the Eclipse Marketplace or when you install by searching in the Eclipse Extensions view.
```

Comment:
I think we should provide more detail about the two parts.

"The first part" -> "The Eclipse plugin"